yt/visualization/plot_window.py

```diff
@@ -2780,3 +2912,38 @@ def plot_2d(
         aspect=aspect,
         data_source=data_source,
     )
+
+
+def _swap_axes_extents(extent):
```

Comment:
This looks like the appropriate signature here, might as well use it (you'll need to import `TypeVar` from the `typing` module)
```suggestion
T = TypeVar("T", tuple, list)

def _swap_axes_extents(extent: T) -> T:
```